contracts/Collector.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./interfaces/ICollector.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+contract Collector is ICollector{
+    IERC20 public token;
+    address public owner;
+    RevenuePartner[] private partners;
```

Comment:
As we already talked about, we could expose the partner addresses only. Not required for the current purpose, but do we need to keep track of it?

Reply:
i have added this to the pending issues list so we can discuss it, i haven't created a JIRA for it since i don't know if we're actually going to move forwards with it